### PR TITLE
Add EchoPilot AI board id

### PR DESCRIPTION
**Pull request by climr**

### CODE REVIEW

**Review by tridge — APPROVED** [review]

File: Tools/AP_Bootloader/board_types.txt [hunk 1]
```diff
@@ -261,6 +261,7 @@ AP_HW_CUBEORANGE_PERIPH              1400
 AP_HW_CUBEBLACK_PERIPH               1401
 AP_HW_PIXRACER_PERIPH                1402
 AP_HW_SWBOOMBOARD_PERIPH             1403
+AP_HW_EchoPilotAI                    1404
```
Comment (tridge) [inline Tools/AP_Bootloader/board_types.txt:264]: apologies, this is failing CI with:
  ValueError: Duplicate ID 1404 in file for (AP_HW_EchoPilotAI) and (AP_HW_VIMDRONES_FLOW)

**Review by tridge — CHANGES_REQUESTED** [review]
sorry, you will need to change your board ID. We were negligent in not merging this earlier.

File: Tools/AP_Bootloader/board_types.txt (hunk 1 above)
Comment (Hwurzburg) [inline Tools/AP_Bootloader/board_types.txt:264]: try 1408

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

**Comment by rmackay9** [comment]
It would be good to also modify the commit message to start with "Tools:" or "Bootloader:".  To be clear, I meant the commit message, no the PR title.

**Comment by peterbarker** [comment]
@climr please open a fresh pull request if you still want to reserve a board ID.  We'll try to be a bit more diligent next time!

**Pull request closed without merging**